src/app/services/products.service.ts

```diff
@@ -0,0 +1,27 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { IProduct } from '@shared/types';
+import { environment as env } from './../../environments/environment.development';
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
providedIn: 'root', it makes sense to leave it only for a global service, for example (modal windows, notifications, etc.)